omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/DrugResource1_10.java

```diff
@@ -52,8 +60,46 @@ public DelegatingResourceDescription getCreatableProperties() {
 		DelegatingResourceDescription description = super.getCreatableProperties();
 		description.addProperty("strength");
 		description.addRequiredProperty("name");
+		description.addProperty("drugReferenceMaps");
 		
 		return description;
 	}
 	
+	public Model getGETModel(Representation rep) {
+		ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation) {
+			modelImpl
+			        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapGetRef"));
+		} else if (rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapGet"));
+		}
+		return modelImpl;
+	}
+	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		return new ModelImpl()
+		        .property("drugReferenceMaps", new RefProperty("#/definitions/DrugreferencemapCreate"));
+	}
+	
+	@PropertySetter("drugReferenceMaps")
```

Comment:
What is the use of this method? Will any test fail without it?

Reply:
Your right, the above has already been taken care of in the Drug class by the drugReferenceMaps setters and getters, so this was kind of a repetition .